users/details/script.js

```diff
@@ -341,6 +346,11 @@ function createSingleTaskCard(task) {
   return container;
 }
 
+function generateReadableDateFromSecondsTimeStamp(timeStamp) {
```

Comment:
- Will `convertSecondsTimeStampToDate` be a more suitable name here as we are not generating anything but rather converting the given timestamp into another format?

Reply:
Thanks for the comment @yesyash.
I named it the way it is because there is already a function called generateReadableDateFromTimeStamp.
So, if someone is working on the code in the future and finds 2 similar functions, they will get to know the difference just by reading the name and won't have to spend unnecessary time going through the code.
I get your point and I can change it if it is required. Do let me know if the above reasoning is valid for the name to be the way it is right now.

Reply:
- Can we not use the function you're talking about instead of creating a new one?

Reply:
(Commenting just for the record)
As we discussed on the call. I have created this function for APIs which have the timestamp in seconds and not in milliseconds.